appinterface/projection/crossfire/crossfire.go

```diff
@@ -569,6 +604,15 @@ func (projection *Crossfire) updateTxSentCount(
 				return fmt.Errorf("error Incrementing tx sent count: %v", errIncrementingTotal)
 			}
 
+			//Increment count for Weekly Jackpot (If Applicable)
+			if len(weeklyJackpotDBKey) <= 0 {
```

Comment:
```suggestion
			if weeklyJackpotDBKey == "" {
```

Reply:
I agree with this. Is it because of being better in performance, Just curious to understand?